models/user/setting_keys.go

```diff
@@ -12,4 +12,6 @@ const (
 	UserActivityPubPrivPem = "activitypub.priv_pem"
 	// UserActivityPubPubPem is user's public key
 	UserActivityPubPubPem = "activitypub.pub_pem"
+	// SettingsKeyProfileRepoName is user's profile repo name
+	SettingsKeyProfileRepoName = "profile_repo_name"
```

Comment:
It's better to use two sections split with a dot symbol.

Reply:
```suggestion
	SettingsKeyProfileRepoName = "profile.repo_name"
```

How about this?